src/core/index.js

```diff
@@ -8,6 +8,8 @@ exports.Aggregation = require('./aggregation');
 
 exports.Query = require('./query');
 
+exports.Knn = require('./knn');
```

Comment:
```suggestion
exports.KNN = require('./knn');
```
Can we name this as `KNN`?